src/components/ec-input-field/ec-input-field.vue

```diff
@@ -96,9 +98,18 @@ export default {
       type: Number,
       default: 20,
     },
+    isInGroup: {
+      type: String,
+    },
+    idFromParent: {
```

Comment:
Why do you need this? The name of the prop indicates it's a code smell and I don't see any purpose.

Reply:
@mcibique maybe there is a better way. I want to connect the currency label with an input field (for-id)

Reply:
Okay now I understand what is this for. Rename this prop to `id`, because every prop is `FromParent`, we don't need that. And then rename the computed prop to `inputId`. 

BTW, In inputId, you can do this:
```
return this.id || `ec-input-field-${this._uid}`;
```